### PR TITLE
feat: use different interest for different no_ of_ completed orders

### PR DESCRIPTION
**Pull request by cheedinmar**

### CODE REVIEW

File: src/utilities/calculator.js [hunk 1]
```diff
@@ -1,4 +1,9 @@
-const calculate = (productPrice, data, params, percentage_discount) => {
+const calculate = (productPrice, data, params, percentage_discount, no_of_orders) => {
+  const interest = {
```
Comment (teebarg) [inline src/utilities/calculator.js:2]: Can we just use a list? [3.5, 3, 2.75]

File: src/views/CreateOrder.vue [hunk 2]
```diff
@@ -326,7 +326,10 @@ function setName(obj) {
                 x.repayment_duration_id == Order.repayment_duration_id
             );
         });
-        const { total, actualDownpayment, rePayment } = calculate(Order.amount, Data, params, 0);
+        const no_of_orders = Customer.value.orders.length
+        console.log(Customer.value.orders.length, 'customer')
```
Comment (teebarg) [inline src/views/CreateOrder.vue:330]: let's remove log

File: src/views/CreateOrder.vue [hunk 3]
```diff
@@ -326,7 +326,10 @@ function setName(obj) {
...
+        const { total, actualDownpayment, rePayment } = calculate(Data.amount, Data, params, 0, no_of_orders > 2 ? 2 : no_of_orders);
+       // const { total, actualDownpayment, rePayment } = calculate(Order.amount, Data, params, 0);
```
Comment (teebarg) [inline src/views/CreateOrder.vue:332]: remove comment

File: src/utilities/calculator.js (hunk 1 above)
Comment (cheedinmar) [reply src/utilities/calculator.js:2]: yes but then it is better to use obj, incase we want to add more interest rate, the keys in that obj represents the length of the existing customer orders

Comment (teebarg) [reply src/utilities/calculator.js:2]: We can update then but a list looks like the right answer for this

**Review by teebarg — APPROVED** [review]

**Pull request merged**